.phpunit.result.cache

```diff
@@ -0,0 +1 @@
+C:37:"PHPUnit\Runner\DefaultTestResultCache":1100:{a:2:{s:7:"defects";a:0:{}s:5:"times";a:13:{s:63:"Yiisoft\Log\Target\File\Tests\FileRotatorTest::testRotateByCopy";d:0.037;s:65:"Yiisoft\Log\Target\File\Tests\FileRotatorTest::testRotateByRename";d:0.001;s:65:"Yiisoft\Log\Target\File\Tests\FileRotatorTest::testRotateMaxFiles";d:0.003;s:68:"Yiisoft\Log\Target\File\Tests\FileRotatorTest::testRotateMaxFileSize";d:0.005;s:69:"Yiisoft\Log\Target\File\Tests\FileRotatorTest::testDefaultMaxFileSize";d:0;s:74:"Yiisoft\Log\Target\File\Tests\FileRotatorTest::testMaxFileSizeLowerThanOne";d:0.009;s:77:"Yiisoft\Log\Target\File\Tests\FileRotatorTest::testSetMaxFileSizeLowerThanOne";d:0;s:66:"Yiisoft\Log\Target\File\Tests\FileRotatorTest::testDefaultMaxFiles";d:0;s:71:"Yiisoft\Log\Target\File\Tests\FileRotatorTest::testMaxFilesLowerThanOne";d:0;s:74:"Yiisoft\Log\Target\File\Tests\FileRotatorTest::testSetMaxFilesLowerThanOne";d:0;s:54:"Yiisoft\Log\Target\File\Tests\FileTargetTest::testInit";d:0;s:65:"Yiisoft\Log\Target\File\Tests\FileTargetTest::testExportEmptyFile";d:0.001;s:64:"Yiisoft\Log\Target\File\Tests\FileTargetTest::testExportMessages";d:0;}}}
```

Comment:
Should not be committed and should be added to `.gitignore`.

Reply:
I'm sorry, I didn't look.